docs/design-docs/reporting-sot.md

```diff
@@ -0,0 +1,538 @@
+
+Existing reporting views were created on an ad-hoc basis to meet the needs of
+the time. As a result, it's difficult to know if they are perfectly accurate,
+given the necessary complexity of their underlying SQL. Their complexity and
+lack of shared schema also means they're difficult to work on for newcomers.
+
+Consolidating reporting into larger tables would let us focus our debugging and
+documentation efforts into just a few areas, rather than a dispersed set of
+assets.
+
+#### Requirements
+
+New reporting tables should:
+
+- Have strict tests for:
+  - Completeness
+  - Errors
+  - Representativeness (each geography has a row for each possible value)
+- Have tests run daily, regardless of build status
+- Subsume the duties of current reporting tables/views (eventually, over time)
+
+## Design
+
+The main output of this project would be *new reporting tables* in our
+AWS Athena data lakehouse.
+
+Each table would have roughly the same schema/primary key, but a different set
+of statistics. The Data Department spent some time scoping the overall table
+schema in-person and came up with the following:
+
+| method | year | geography_type | geography_id | geography_data_year | group_type     | group_id | stage_name | stage_id | additional_cols... |
+|--------|------|----------------|--------------|---------------------|----------------|----------|------------|----------|--------------------|
+| v1     | 2023 | municipality   | Chicago      | 2022                | class          | 204      | board      | 3        | ...                |
+| v1     | 2010 | triad          | 1            | 2010                | modeling_group | SF       | mailed     | 1        | ...                |
+
+### Primary key columns
+
+- `method (string)` - The method version number used for lifecycle
+  management. Hopefully set to `v1` forever, but more likely to increment for
+  each breaking change
+- `year (string)` - Tax year. Should derive existence from `iasworld.pardat`
+- `geography_type (string)` - The type of geography the stat is grouped by,
+  e.g. municipality, triad, township, etc. Should encompass all geographies
+  currently in our PIN universe
+- `geography_id (string)` - The identifier for each unit of geography,
+  e.g. for municipality this would be `Chicago`, for township this might be
+  "Thornton"
+- `geography_data_year (string)` - The year of the file the geography
+  definition is sourced from e.g. a 2020 Census boundary would be `2020`
+- `group_type (string)` - The sub-group for aggregation within each geometry
+  e.g. class, modeling group, etc.
+- `group_id (string)` - The sub-group identifier e.g. for class it would be
+  `202`, `203`, etc.
+- `(optional) stage_name (string)` - Name of the assessment stage used for
+  grouping, one of `mailed`, `certified`, or `board`
+- `(optional) stage_id (string)` - Numeric identifier of the assessment
+  stage, one of `1` (mailed), `2` (certified), `3` (board)
+
+#### Enumerated geography_type
```

Comment:
This is great! My only note is a complication about `ward_num`; because we have ward nums for two municipalities (Chicago and Evanston), historically, I've asked for the municipality name to be appended with `ward_num` in the `geo_id `(for example, "Chicago 1").